### DISCUSSION TITLE
Any chance for ST7701S support? (square 4" TFT)

### DISCUSSION THREAD

**Discussion by XFer012** [olikraus/u8g2#2077] [Q&A] [answered]
Hello,
I could not find ST7701(s) listed among the support displays.
It's a RGB TFT, square instead of rectangular (480x480).
Its interface is SPI (3-wire) + RGB, total required pins = 32(!).
Any chance it may be supported in the future?
Thanks!

**Comment by olikraus** [comment, marked as answer]
U8g2 is a library for monochrome displays only
